tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -222,6 +224,108 @@ def test_non_org_rename_403(self):
         self.get_error_response(other_org.slug, "old_slug", status_code=403)
 
 
+class ProjectUpdateTestTokenAuthenticated(APITestCase):
+    endpoint = "sentry-api-0-project-details"
+    method = "put"
+
+    def setUp(self):
+        super().setUp()
+        self.project = self.create_project(platform="javascript")
+        self.user = self.create_user("bar@example.com")
+
+    def test_member_updates_denied_with_token(self):
+        self.create_member(
+            user=self.user,
+            organization=self.project.organization,
+            teams=[self.project.teams.first()],
+            role="member",
+        )
+
+        token = ApiToken.objects.create(user=self.user, scope_list=["project:write"])
```

Comment:
@mdtro Do you know if we also need to worry about `org:write` scope (and possibly other write scopes) as well?

Reply:
I manually checked to see if I could edit Organization level properties as a _member_ with an API token that had all permissions checked. It was not successful. 

I can add in some tests in a separate PR for that if necessary.